pkg/types/ibmcloud/validation/machinepool.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // ValidateMachinePool validates the MachinePool.
-func ValidateMachinePool(mp *ibmcloud.MachinePool, path *field.Path) field.ErrorList {
+func ValidateMachinePool(platform *ibmcloud.Platform, mp *ibmcloud.MachinePool, path *field.Path) field.ErrorList {
```

Comment:
What are these validation changes doing? I don't see anything other than passing in an additional parameter.

Reply:
Yep, good point. Removed the changes since they are not being used here.